integrationtests/tests/tests.rs

```diff
@@ -1079,6 +1083,8 @@ async fn rejoin_consensus_single_peer() -> Result<()> {
 
         let new_height = client2.await_consensus_block_height(height).await.unwrap();
         assert_eq!(new_height, height);
+        // TODO: find out why this is flaky
+        // fed.assert_peers_synchronized().await;
```

Comment:
These tests intentionally have peers fall behind in epochs.

Reply:
:thinking: 
But are they not coming back?